content/_build/workers.html

```diff
@@ -73,13 +73,13 @@
 Finally, after killing the anaconda build worker process, it is required to deregister the worker, unless you plan to start the worker again with the same worker id and configuration.  The deregister step can be done by giving the configuration file that was an --output of the anaconda build register step:
 
 {% syntax bash %}
-    anaconda build deregister --config worker.yaml
+    anaconda build deregister --worker-id <worker-id-from-register-step>
```

Comment:
@PeterDSteinberg in the future this could also be just:

`anaconda build deregister <worker_id>`